Subproject2/1. SOVA/Controllers/PostController.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace _1._SOVA.Controllers
+{
+    public class PostController
+    {
+    }
+}
```

Comment:
We don't have need `PostController` as we don't have anything called Post. We have Submissions, Questions, Answers and Comments.

Reply:
either delete this controller or update it with some other necessary controller.